docs/casengines.md

```diff
@@ -1,172 +1,153 @@
 
-3. **OpenEBS Local PV** - OpenEBS Local PV is a new storage engine that can create persistent volumes or PVs out of local disks or host paths on the worker nodes. This CAS engine is available from 1.0.0 version of OpenEBS. With OpenEBS Local PV, the performance will be equivalent of either the local disk or the file system (host path) on which the volumes are created. Many cloud native applications may not require advanced storage features like replication or snapshots or clones as they themselves provide these features. Such applications require access to a managed disks as persistent volumes. Read more details of OpenEBS Local PV [here](/docs/next/localpv.html)
+## Stateful Workload Capabilities
 
+State is an integral part of any application often times, used without realizing that it actually exists. Examples of Stateful Applications include SQL/NoSQL Database, Object and Key/Value stores, Message Bus, Storage systems optimized to store Logs and Metrics, Code/Container/Configuration Repositories, and many more. Each stateful applications comes with a certain capabilities and depends on the storage for complimentary capabilities. For example:
+- Stateful workloads like MongoDB have availability features like protecting against node failures built into them. Such systems will expect the Data engines to provide capacity and performance required with the data consistency/durability at the block level.
+- Stateful workloads like Cassandra can benefit from the availability features from the data engines as it might help speed up the rebuild times required to rebuild a failed cassandra node. However this comes at the cost of using extra storage by the data engines. 
+- With serverless and cloud native becoming mainstream a key shift has been in terms of the capacity required and the duration for which that capacity may be required. These are applications that are launched as part of the DevSecOps pipelines where a PostgreSQL or other types of systems may be launched to run for a few minutes, hours or days and then the resulting information is saved in another storage system and this instance is itself deprovisioned. 
 
+Depending on the Stateful workload capabilities, the recommended engines are:
+| Stateful Workload Capabilities                       | Jiva    |  cStor   |  Mayastor  | LocalPV  | 
+| -----------------------------------------------------| :---:   | :------: | :--------: | :------: |
+| Pod can be pinned to a single node                   | Yes     |   Yes    | Yes        | Yes      |
+| Pod are short lived and can be restarted on same node| Yes     |   Yes    | Yes        | Yes      |
+| Pod when moved can rebuild its data                  | Yes     |   Yes    | Yes        | Yes      |
+| Pod when moved to new node, can't rebuild its data   | Yes     |   Yes    | Yes        | No       |
+| Pod need resiliency against device faults/error      | Yes     |   Yes    | Yes        | No*      |
```

Comment:
We can move No* one space to the right so that it will be aligned to other texts in the same column.